VENDORING.md

```diff
@@ -0,0 +1,43 @@
+# Vendoring policies
+
+This document outlines recommended Vendoring policies for Docker repositories.
+(Example, libnetwork is a Docker repo and logrus is not.)
+
+## Vendoring using tags
+
+Commit ID based vendoring provides little/no information about the updates
+vendored. To fix this, vendors will now require that repositories use annotated
+tags to snapshot commits.
+
+Each tag should:
```

Comment:
As an alternative, github makes a solid change log in the releases link: https://github.com/docker/distribution/releases.